src/data/pokedex_orders.h

```diff
@@ -1,1609 +1,1609 @@
-    282,  // Gardevoir
-    210,  // Granbull
-    106,  // Hitmonlee
-    107,  // Hitmonchan
-    227,  // Skarmory
-    331,  // Cacnea
-    257,  // Blaziken
-    254,  // Sceptile
-    336,  // Seviper
-    366,  // Clamperl
-    145,  // Zapdos
-    214,  // Heracross
-     62,  // Poliwrath
-    122,  // Mr. mime
-    127,  // Pinsir
-    272,  // Ludicolo
-     73,  // Tentacruel
-     42,  // Golbat
-    219,  // Magcargo
-    144,  // Articuno
-    221,  // Piloswine
-    123,  // Scyther
-     64,  // Kadabra
-    235,  // Smeargle
-    142,  // Aerodactyl
-    275,  // Shiftry
-     99,  // Kingler
-     31,  // Nidoqueen
-     82,  // Magneton
-    304,  // Aron
-    381,  // Latios
-    146,  // Moltres
-    346,  // Cradily
-    386,  // Deoxys
-    181,  // Ampharos
-     34,  // Nidoking
-    207,  // Gligar
-     24,  // Arbok
-    108,  // Lickitung
-    101,  // Electrode
-    348,  // Armaldo
-     67,  // Machoke
-    234,  // Stantler
-    326,  // Grumpig
-    246,  // Larvitar
-    169,  // Crobat
-    195,  // Quagsire
-    241,  // Miltank
-     97,  // Hypno
-     55,  // Golduck
-    332,  // Cacturne
-     80,  // Slowbro
-    157,  // Typhlosion
-    199,  // Slowking
-    115,  // Kangaskhan
-    121,  // Starmie
-    317,  // Swalot
-    324,  // Torkoal
-    260,  // Swampert
-    330,  // Flygon
-    295,  // Exploud
-     85,  // Dodrio
-      9,  // Blastoise
-    296,  // Makuhita
-    364,  // Sealeo
-    128,  // Tauros
-    319,  // Sharpedo
-    160,  // Feraligatr
-     86,  // Seel
-      6,  // Charizard
-     78,  // Rapidash
-    374,  // Beldum
-    299,  // Nosepass
-      3,  // Venusaur
-    357,  // Tropius
-    154,  // Meganium
-    373,  // Salamence
-     75,  // Graveler
-    344,  // Claydol
-    372,  // Shelgon
-    111,  // Rhyhorn
-    212,  // Scizor
-     87,  // Dewgong
-    112,  // Rhydon
-    232,  // Donphan
-    103,  // Exeggutor
-    305,  // Lairon
-    150,  // Mewtwo
-    217,  // Ursaring
-    205,  // Forretress
-     68,  // Machamp
-    320,  // Wailmer
-    289,  // Slaking
-     91,  // Cloyster
-    365,  // Walrein
-    247,  // Pupitar
-    230,  // Kingdra
-    338,  // Solrock
-     59,  // Arcanine
-    350,  // Milotic
-    337,  // Lunatone
-    378,  // Regice
-    243,  // Raikou
-    245,  // Suicune
-    244,  // Entei
-    250,  // Ho-Oh
-    248,  // Tyranitar
-    375,  // Metang
-    379,  // Registeel
-    384,  // Rayquaza
-     95,  // Onix
-    149,  // Dragonite
-    249,  // Lugia
-    131,  // Lapras
-    323,  // Camerupt
-    226,  // Mantine
-    377,  // Regirock
-    130,  // Gyarados
-    297,  // Hariyama
-    362,  // Glalie
-     76,  // Golem
-    382,  // Kyogre
-    306,  // Aggron
-    321,  // Wailord
-    208,  // Steelix
-    143,  // Snorlax
-    376,  // Metagross
-    383,  // Groudon
+	NATIONAL_DEX_GASTLY,  // Gastly
+	NATIONAL_DEX_HAUNTER,  // Haunter
+    NATIONAL_DEX_HOPPIP,  // Hoppip
+	NATIONAL_DEX_DIGLETT,  // Diglett
```

Comment:
there is a mix of tabs and 4 spaces throughout this entire file--use 4 spaces for all indentation.